book/glossary/glossary.tex

```diff
@@ -0,0 +1,374 @@
+of dough like a balloon until it bursts. The resulting chart, or ``alveogram'',
+displays a curve that represents the balance between the dough's elasticity and
+extensibility. Specific parameters derived from the curve, such as the P (pressure
+required to inflate the dough) and L (extensibility of the dough), provide invaluable
+insights to bakers and millers regarding the flour's potential performance in
+bread-making. By analyzing the alveogram, professionals can make informed decisions
+about the suitability of a flour for certain baking applications, as well as
+potential blending needs with other flours.
+
+\textbf{Alveoli (sg: Alveolus):} The little pockets that form the crumb, formed by
+the gluten matrix trapping carbon dioxide.
+
+\textbf{Alpha-amylase:} A type of amylase that breaks down starch molecules into
+shorter fragments, producing maltose and some glucose.
+
+\textbf{All purpose flour:} A general flour that’s balanced to make breads and also
+cakes. In germany this is type 550.
+
+\textbf{Amylase:} An enzyme that breaks down starches into simpler sugars, facilitating
+the fermentation process in beer and bread making. When making beer the temperature
+of the brew is kept for extended periods at certain temperatures to ensure that most
+starches are broken down to sugars. These sugars are then consumed by the microbes
+during the fermentation process.
+
+\textbf{Autolyse:} A process where flour and water are mixed and then left to rest
+before adding other ingredients. This activates enzymes such as amylase and protease.
+By doing so the bulk fermentation time is shortened and the final loaf will have
+better properties. The browning of the loaf becomes better and the crumb fluffier.
+An autolyse is recommended when using a high percentage of starter to inoculate the
+dough (> 20\%). An alternative easier approach can be the fermentolyse.
+
+\textbf{Baker’s math:} Baker’s math is a ratio based system of sharing recipes,
+making them easily scalable. It’s based on the total weight of the flour in a formula,
+where each ingredients weight is divided by the flours weight to give a percentage.
+For 500 grams of flour you could be using 60\% of water (300 grams), 10\% of starter
+(50 grams) and 2\% of salt (10 grams).
+
+\textbf{Baker’s percentage:} See Baker’s math.
+
+\textbf{Baking:} The final, transformative step in bread making wherein dough is
+exposed to high temperatures, causing a series of chemical and physical reactions
+that result in a finished loaf of bread. During the baking stage:
+
+1: \textit{Yeast Activity \& Oven Spring:} In the initial phase of baking, the
+temperature inside the dough rises, increasing yeast activity. This results in rapid
+carbon dioxide production, leading to what bakers refer to as ``oven spring'', or the
+rapid rise of the loaf.
+
+2: \textit{Protein Coagulation:} As the temperature continues to climb, the proteins
```

Comment:
No manual numbering, use enumerate instead

Reply:
Didn't work with description. For this reason I left it the way it is for now. Feel fee to improve.